examples/hitl/rearrange_v2/README.md

```diff
@@ -33,6 +47,11 @@ Headless server:
 python examples/hitl/rearrange_v2/rearrange_v2.py +experiment=headless_server
 ```
 
+Testing Habitat-LLM episodes using a headed server:
+```bash
+python examples/hitl/rearrange_v2/rearrange_v2.py --config-name rearrange_v2_humanoid_only habitat_hitl.networking.enable=True
+```
+
```

Comment:
Can you add instructions for testing Habitat-llm episodes with a user-controlled humanoid and user-controlled spot? @0mdc will need this.

Reply:
Added command to test it with user-controlled humanoid. For humanoid+spot I'll add it with habitat-llm policy changes